warcit/warcit.py

```diff
@@ -40,26 +40,63 @@ def main(args=None):
     parser.add_argument('-d', '--fixed-dt',
                         help='''Set resource date and time in YYYYMMDDHHMMSS format.
                                 If not given, last modified date of files is used.''',
-                        metavar='timestamp')
+                        metavar='<TIMESTAMP>')
 
     parser.add_argument('-n', '--name',
-                        help='''base name for WARC file''',
+                        help='''Base name for WARC file, appropriate extension will be
+                                added automatically.''',
                         metavar='name')
 
     parser.add_argument('-a', '--append', action='store_true')
     parser.add_argument('-o', '--overwrite', action='store_true')
+   
+
+    parser.add_argument('--use-magic', '--magic',
```

Comment:
It seems like this is redundant with `--charset` below. Would recommend removing, or keeping as boolean to only enable 'magic' detection.

Reply:
This is not redundant, charset and file type detection are separated.

Reply:
Ok, yes, makes sense now. Maybe should also have `--mime` flag to make it more clear?